iceoryx_hoofs/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
-# Copyright (c) 2019 - 2021 by Robert Bosch GmbH. All rights reserved.
-# Copyright (c) 2020 - 2022 by Apex.AI Inc. All rights reserved.
+# Copyright (c) 2019 - 2021 by Robert Bosch GmbH. All rights reserved. Copyright
```

Comment:
The copyright header should not be reformatted.

Reply:
I think I will need to fence this block from formatting, so disabling the formatter before it and then enable it after